src/index.ts

```diff
@@ -7061,7 +7061,7 @@ const shardusSetup = (): void => {
       const currentlyActivatedNode = currentCycle.activated.includes(nodeId)
       if (currentlyActivatedNode) return
 
-      if (eventType === 'node-activated') {
+      if (eventType === 'node-activated' && AccountsStorage.cachedNetworkAccount?.current.utilityFlags.enableRewardTXs) {
```

Comment:
just to call something out, we need to decide what we want to happen here if cachedNetworkAccount is undefined. as it is, if it is not defined, then reward txs will effectively be turned off. is this what we want, or should they work if the cached network account is undefined?

Reply:
please make new commits on top of dev.  we don't need to revert the revert

Reply:
This is just the quick fix.  we sill need to figure out why cached network account is not available here.

Also we need to figure out if it is not available for an indefinite amount of time.

Reply:
this PR can be on hold until we find the core fix.

Reply:
I think that AccountsStorage.cachedNetworkAccount being undefined is fatal.   we have to solve that

Reply:
oh i think i know the reason. we try to fetch it from the archiver, but since that query fails, we use default configs. this leads to the cached network account being undefined until the init network tx. I looked into the archiver code somewhat, and it looks like its undefined at the time we query it

Reply:
as a bonus, if we can fix the fecth from archiver, then network start up will be 1-2 minutes faster

Reply:
lmk if you want me to look into it more

Reply:
please give that fix a test and see if it works out

Reply:
i dont have a fix for the archiver ready. ill try to find one